Core/clim-basic/recording.lisp

```diff
@@ -396,13 +394,11 @@ recording stream. If it is T, *STANDARD-OUTPUT* is used.")
          ;; out shortly after being drawn. So, we aren't ready for
          ;; this yet.  ..Actually, it isn't necessarily
          ;; faster. Depends on the app.
-         #+NIL
+         #+ (or)
```

Comment:
Do we really want to keep this around?

Reply:
I think that using `#+nil` is a bad practice becase it is wrong from semantical point of view, so I'd say yes, `#+(or)` is the only proper way to comment block of code.

Reply:
ah, you mean call to `queue-repaint`. I think that `queue-repaint` is the right thing here, so if we get acceptably faster at some point, I'd want to make it the default.